apps/web/src/components/docs/Docs.tsx

```diff
@@ -157,7 +157,17 @@ export const Docs = ({ path = '', children, actions }: { path?: string; children
             },
             Info: ({ className, ...props }: any) => {
               return (
-                <Alert style={{ borderRadius: 12 }} color="gray" {...props} icon={<IconInfoOutline color="white" />} />
+                <Alert
+                  className={css({
+                    borderRadius: '75',
+                    backgroundColor: 'mauve.60.dark !important',
```

Comment:
All `!important` I have added in latest commit was needed to affect style of their components

Reply:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** While using the "new" base color tokens is better than using the `legacy.*` tokens, we should still be aiming to use semantic tokens as much as possible in application code so that we can better handle dark/light and in a semantic way.‏